### PR TITLE
Feat(event) - added nonInteraction option

### PR DESCRIPTION
**Pull request by adlenafane**
Implementing the non-interaction events https://developers.google.com/analytics/devguides/collection/analyticsjs/events#non-interaction_events
@kaesonho I'm not sure about how you would like to implement this. Could you review please?

### CODE REVIEW

**Comment by hero78119** [comment]
IMO the flatten-style arguments for ga is a little bit awkward :(
It strictly depends on the arguments order, which might be lead to trouble of scalability  
According to [fieldsObject](https://developers.google.com/analytics/devguides/collection/analyticsjs/events#examples), it should be better to refactor to  sth like

``` js
var params = {
    hitType: 'event',
    eventCategory: model.category || DEFAULT_CATEGORY,
    eventAction: model.action || DEFAULT_ACTION,
    eventLabel: model.label || i13nNode.getText(payload.target) || DEFAULT_LABEL,
    eventValue: model.value || 0,
    transport: model.transport || 'none',
    nonInteraction: model.nonInteraction || false
};

_command.call(this, {
  tracker: model.tracker || '',
  commandName: 'send',
  arguments: [params]
});
```

File: index.client.js [hunk 1]
```diff
@@ -91,6 +91,11 @@ ReactI13nGoogleAnalytics.prototype.getPlugin = function () {
                     if (model.value) {
                         params.push(model.value);
                     }
+                    if (model.nonInteraction) {
```
Comment (kaesonho) [inline index.client.js]: yeah will something be broken if value doesn't exist but nonInteraction does?

Comment (kaesonho) [inline index.client.js]: @adlenafane  hi any update for this?

**Comment by adlenafane** [comment]
@hero78119 @kaesonho I did a bit of refacto to use the ga fieldsObject :)

**Comment by kaesonho** [comment]
thanks for the update 👍

**Comment by kaesonho** [comment]
it's published https://github.com/kaesonho/react-i13n-ga/releases/tag/v0.2.5 :)

**Pull request merged**